src/libraries/System.Net.Http/src/System/Net/Http/DiagnosticsHandlerLoggingStrings.cs

```diff
@@ -9,12 +9,15 @@ namespace System.Net.Http
     internal static class DiagnosticsHandlerLoggingStrings
     {
         public const string DiagnosticListenerName         = "HttpHandlerDiagnosticListener";
-        public const string Namespace                      = "System.Net.Http";
-        public const string RequestWriteNameDeprecated     = Namespace + ".Request";
-        public const string ResponseWriteNameDeprecated    = Namespace + ".Response";
-        public const string ExceptionEventName             = Namespace + ".Exception";
-        public const string ActivityName                   = Namespace + ".HttpRequestOut";
-        public const string ActivityStartName              = ActivityName + ".Start";
-        public const string ActivityStopName               = ActivityName + ".Stop";
+        public const string RequestNamespace               = "System.Net.Http";
+        public const string RequestWriteNameDeprecated     = RequestNamespace + ".Request";
+        public const string ResponseWriteNameDeprecated    = RequestNamespace + ".Response";
+        public const string ExceptionEventName             = RequestNamespace + ".Exception";
+        public const string RequestActivityName            = RequestNamespace + ".HttpRequestOut";
+        public const string RequestActivityStartName       = RequestActivityName + ".Start";
+        public const string RequestActivityStopName        = RequestActivityName + ".Stop";
+
+        public const string ConnectionNamespace            = "System.Net.Http.Connections";
+        public const string ConnectionActivityName = ConnectionNamespace + ".HttpConnectionOut";
```

Comment:
Is it a convention to have an `Out` postfix on these names?

Reply:
There is a .NET convention to do so, but not an OTel convention. In general we probably want to sync with @lmolkova on naming and conventions here. If we can establish an OTel convention that would be preferable.

Reply:
AFAIK this one is OperationName and I think has no OTel equivalent. 
For the `DisplayName`, we'd need to define a new span and span name. 

Does this one last for the connection lifetime? If so I'd use something like

`connection {server.address}:{server.port}`

Reply:
I am trying to figure out what is more clear in a trace:
`System.Net.Http.Connections.HttpConnectionOut`
or
`System.Net.Http.Connections.HttpConnection` ?

Reply:
> AFAIK this one is OperationName and I think has no OTel equivalent.
For the DisplayName, we'd need to define a new span and span name.

OperationName is the default value used for DisplayName unless it is overriden somewhere. I don't think we get any value picking one string for OperationName and a different string for DisplayName. So we probably want OperationName = DisplayName = connection {server.address}:{server.port}

> I am trying to figure out what is more clear in a trace

What gets transmitted by OTLP and rendered in the UI should be DisplayName and @lmolkova is suggesting that be "connection {server.address}:{server.port}". I think that would be clearer than either of those namespaced names no?

Reply:
OperationName could still be important for the DiagnosticSource (DS -> EventSource) as the event name and then it's nice to have a const string for it, but you all know much more about it than I am.

Reply:
> OperationName could still be important for the DiagnosticSource

This PR is not adding `DiagnosticSource` / `DiagnosticListener`, only `ActivitySource` + activities, because this is what #93832 is about in my understanding. Is there a strong reason for also adding a `DiagnosticListener` like we have it in `DiagnosticHandler`? I know little about how is this being consumed.

> So we probably want OperationName = DisplayName = connection {server.address}:{server.port}

Given that OperationName has no OTel equivalent according to https://github.com/dotnet/runtime/pull/101814#discussion_r1591774341, IMO it would be better if the OperationName followed the existing .NET convention for consistency. We can use a different DisplayName if we want. In that case wouldn't we also want to change the DisplayName for the request activity, or do we consider it a breaking change we better avoid?

Reply:
> OperationName could still be important for the DiagnosticSource (DS -> EventSource) as the event name and then it's nice to have a const string for it, but you all know much more about it than I do.

You can filter on Activity name as part of DiagnosticSourceEventSource and ActivityListener.Start/Stop, but not as part of ActivityListener.Sample or OTel. Given that OTel is a primary scenario we want to support I'd want to ensure that ActivitySource alone is sufficiently fine-grained to do the filtering we'd like to do. It won't harm anything to pick some constant value for OperationName that differs from DisplayName, but ideally nothing should be relying on us doing that and we'd move away from showing OperationName in our tools as a distinct piece of data.